src/lib/npm/index.ts

```diff
@@ -0,0 +1,38 @@
+import fsPromises from 'fs/promises'
+import { z } from 'zod'
+
+
+const packageParser = z.object({
+    version: z.string()
+})
+
+export default class NPMPackageHandler {
```

Comment:
This class needs unit tests similar to what we've done for cargo. Also there's some random whitespace here that I'm surprised the linter is happy with. Might want to have a look at that